shell/browser/electron_download_manager_delegate.h

```diff
@@ -41,7 +41,7 @@ class ElectronDownloadManagerDelegate
   // Get the save path set on the associated api::DownloadItem object
   void GetItemSavePath(download::DownloadItem* item, base::FilePath* path);
   void GetItemSaveDialogOptions(download::DownloadItem* item,
-                                file_dialog::DialogSettings* settings);
+                                file_dialog::DialogSettings* options);
```

Comment:
hypernit: imo `settings` is the better name here as it matches the type name.

Reply:
agreed, we could change the body instead

Reply:
Yea, there were a dozen or so cases like that I skipped over because it would require changing more than just the header, but this one slipped in. There's also the fun case where an `override` has a different name in the definition and the declaration, and neither match the name used in the original declaration that's being overriden. :face_with_head_bandage: